README.rst

```diff
@@ -50,7 +50,8 @@ DNS lookup...
   the lookup.
 * If ``gethostbyname`` does not have it cached then a request is made to the
   known DNS server that was given to the network stack. This is typically the
-  local router or the ISP's caching DNS server
+  local router or the ISP's caching DNS server.
+* Port 53 is opened to send UDP request to DNS server.
```

Comment:
If I understand correctly, DNS requests can become TCP only in case of request like zone transfers which are server-server (replication). It's very unlikely for browser to cause that.